src/languages/en-US/commands/general.json

```
@@ -33,7 +33,7 @@
 	},
 	"helpDm": "📥 | The list of commands you have access to has been sent to your DMs.",
 	"helpNodm": "{{REDCROSS}} | You have DMs disabled so I couldn't send you the list of commands.",
-	"infoBody": "Skyra {{VERSION}} is a multi-purpose Discord Bot designed to run the majority of tasks with a great performance and constant 24/7 uptime.\nShe is built on top of Sapphire Framework, a 'plug-and-play' framework built on top of the Discord.js library.\n\nSkyra features:\n• Advanced Moderation with temporary actions included\n• Announcement management\n• Fully configurable\n• Message logs, member logs, and mod logs\n• Multilingual\n• Profiles and levels, with leaderboards and social management\n• Role management\n• Weeb commands (+10)!\n• A full PokéDex with all data you will need\nAnd more!",
+	"infoBody": "Skyra {{VERSION}} is a multi-purpose Discord Bot designed to run the majority of tasks with a great performance and constant 24/7 uptime.\nShe is built on top of [Sapphire Framework](https://sapphirejs.dev), a 'plug-and-play' framework built on top of the [Discord.js library](https://discord.js.org).\n\nSkyra features:\n• Advanced Moderation with temporary actions included\n• Announcement management\n• Fully configurable\n• Message logs, member logs, and mod logs\n• Multilingual\n• Profiles and levels, with leaderboards and social management\n• Role management\n• Weeb commands (+10)!\n• A full Pokédex with all data you will need\nAnd more!\n\nYou can also find our Privacy Policy [here](https://skyra.pw/privacy).",
```

```suggestion
	"infoBody": "Skyra {{VERSION}} is a multi-purpose Discord Bot designed to run the majority of tasks with great performance and constant 24/7 uptime.\nShe is built on top of [Sapphire Framework](https://sapphirejs.dev), a 'plug-and-play' framework built on top of the [Discord.js library](https://discord.js.org).\n\nSkyra features:\n• Advanced Moderation with temporary actions included\n• Announcement management\n• Fully configurable\n• Message logs, member logs, and mod logs\n• Multilingual\n• Profiles and levels, with leaderboards and social management\n• Role management\n• Weeb commands (+10)!\n• A full Pokédex with all data you will need\nAnd more!\n\nYou can also find our Privacy Policy [here](https://skyra.pw/privacy).",
```

(and for the rest of the languages)

---

No need to change the other languages, because they're not in English, so they don't have this grammar error.

---

that's a very good point

---

Also crowdin is gonna wipe the other languages when the source is edited 🤡